\hypertarget{class_erro_de_logica_exception}{}\doxysection{Referência da Classe Erro\+De\+Logica\+Exception}
\label{class_erro_de_logica_exception}\index{ErroDeLogicaException@{ErroDeLogicaException}}


{\ttfamily \#include $<$excecoes.\+hpp$>$}



Subclasse de std\+::exception.

\doxysubsection*{Membros Públicos}
\begin{DoxyCompactItemize}
\item 
virtual const char $\ast$ \mbox{\hyperlink{class_erro_de_logica_exception_ac375847c2fe34e2a1826061d855f0482}{what}} () const noexcept
\end{DoxyCompactItemize}


\doxysubsection{Funções membros}
\mbox{\Hypertarget{class_erro_de_logica_exception_ac375847c2fe34e2a1826061d855f0482}\label{class_erro_de_logica_exception_ac375847c2fe34e2a1826061d855f0482}} 
\index{ErroDeLogicaException@{ErroDeLogicaException}!what@{what}}
\index{what@{what}!ErroDeLogicaException@{ErroDeLogicaException}}
\doxysubsubsection{\texorpdfstring{what()}{what()}}
{\footnotesize\ttfamily virtual const char $\ast$ Erro\+De\+Logica\+Exception\+::what (\begin{DoxyParamCaption}{ }\end{DoxyParamCaption}) const\hspace{0.3cm}{\ttfamily [inline]}, {\ttfamily [virtual]}, {\ttfamily [noexcept]}}


\begin{DoxyCode}{0}
\DoxyCodeLine{74         \{}
\DoxyCodeLine{75             \textcolor{keywordflow}{return} \textcolor{stringliteral}{"{}Procedimento inesperado."{}};}
\DoxyCodeLine{76         \}}

\end{DoxyCode}


A documentação para essa classe foi gerada a partir do seguinte arquivo\+:\begin{DoxyCompactItemize}
\item 
include/\mbox{\hyperlink{excecoes_8hpp}{excecoes.\+hpp}}\end{DoxyCompactItemize}
